### DISCUSSION TITLE
I would like to see who reacts

### DISCUSSION THREAD

**Discussion by tw4** [community/community#17165] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello to everyone,
It might be better to be able to see who is reacting, rather than showing how many people are responding to the content in the feed in the same way.

**Comment by siminapasat** [comment, marked as answer]
Thanks @mertovich for your feedback. Can you please reiterate what you mean by 

> showing how many people are responding to the content in the feed in the same way.

**Reply by tw4** [reply]
Of course, when we react to content on the Following and For You pages, we see how many people react in total.
current behavior :
![Screen Shot 2022-06-08 at 00 05 42](https://user-images.githubusercontent.com/88425310/172482801-359c9fd0-e427-44df-9b22-57f50bcb11fd.png)

suggested behavior:
![Screen Shot 2022-06-08 at 00 10 58](https://user-images.githubusercontent.com/88425310/172483413-14009899-849a-404b-bfed-b8f442c51cf7.png)

**Reply by siminapasat** [reply]
@mertovich where do you see this UI? Can you please share the URL or a bit more context? It doesn't look to me like it's a screenshot from the feed. Thanks!

**Reply by tw4** [reply]
URL of the place where I see the behavior I want on the feed: https://github.com/ppy/osu/releases/tag/2022.605.1

https://user-images.githubusercontent.com/88425310/172530835-2ccbadb5-8df9-48c5-8ac3-14fd62005dda.mov

**Reply by siminapasat** [reply]
I stand corrected, my team also notified me that can appear on the feed for release type cards. Thanks for your help @mertovich . We'll look into it, especially from the privacy aspect 👍
